src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -244,7 +246,7 @@ public async Task<(Error, IEnumerable<GitHubUser>)> GetUsersByCommit(string owne
                             {
                                 Id = node.author.user?.databaseId,
```

Comment:
are there any case where `author.user` is null?
```suggestion
                                Id = node.author.user.databaseId,
```